CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to wolfi-dev/os
+
+* [Setup development environmment](#setup-development-environment)
```

Comment:
Complete the TOC here, you can use https://github.com/kubernetes-sigs/mdtoc to generate it automatically.